router/routes.js

```diff
@@ -0,0 +1,312 @@
+  try {
+    const value = req.user.id;
+    const updated = await CLient.update(
+      {
+        first_name: req.body.first_name,
+        last_name: req.body.last_name,
+      },
+      { where: { id: value } }
+    );
+    const user = await CLient.findOne({ where: { id: value } });
+    const msg = {
+      to: user.email,
+      from: process.env.SENDER_EMAIL,
+      subject: 'User Updated',
+      text: `Your first-name and last-name has been updated to ${req.body.first_name} and ${req.body.last_name} respectively`,
+      html: `<strong>Your first-name and last-name has been updated to ${req.body.first_name} and ${req.body.last_name} respectively</strong>`,
+    };
+    await sgMail.send(msg).then(() => {
+      console.log('email sent');
+    });
+    res.send({
+      status: '1',
+      message: 'success',
+      data: {
+        message: 'user updated',
+      },
+    });
+  } catch (error) {
+    res.send({
+      status: '0',
+      message: 'failure',
+      data: {
+        message: error,
+      },
+    });
+  }
+});
+// route to delete user
+router.put('/delete', verify, async (req, res) => {
+  try {
+    const value = req.user.id;
+    const deleted = await CLient.destroy({ where: { id: value } });
+    if (!deleted) return res.send('user could not be deleted');
+    res.send({
+      status: '1',
+      message: 'success',
+      data: {
+        message: 'user deleted',
+      },
+    });
+  } catch (error) {
+    res.send({
+      status: '0',
+      message: 'failure',
+      data: {
+        message: error,
+      },
+    });
+  }
+});
+// route to display users
+router.get('/list/:page', async (req, res) => {
+  try {
+    const page = parseInt(req.params.page);
+    if (page < 0) return res.send('please enter a valid page value');
+    const skip = (page - 1) * 10;
+    const data = await CLient.findAll({ limit: 10, offset: skip });
+    res.send({
+      status: '1',
+      message: 'success',
+      data: data,
+    });
+  } catch (error) {
+    res.send({
+      status: '1',
+      message: 'failure',
+      data: error,
+    });
+  }
+});
+
+router.post('/address',verify ,async (req, res) => {
+  try {
+    const address =  await Address.create({
+        userId : req.user.id,
+        address : req.body.address,
+        city : req.body.city,
+        state : req.body.state,
+        pin_code : req.body.pin_code,
+        phone_no : req.body.phone_no
+    })
+    console.log(address)
+    res.send({
+        status : '1',
+        message : 'success',
+        data : {
+            message : 'Address added to user '
+        }
+    })
+  } catch (error) {
+    console.log(error);
+    res.send({
+        status : '0',
+        message : 'failure',
+        data : {
+            message : error
+        }
+    });
+  }
+});
+
+router.get('/get',verify, async (req, res) => {
+  try {
+    const user =await CLient.findAll({include : {model : Address ,as : "Address"}})
+    res.send(user);
```

Comment:
make a unique response pattern
don't send response in different way.

like :
```
res.send(user)
res.send ({status: "", message: "", data: ""})
```
these will give response in different format 
so make then same 

also do same for error too